config/locales/jobs.yml

```diff
@@ -71,35 +71,43 @@ en:
           <span class='govuk-!-font-weight-bold text-wrap-apostrophe'>%{keyword}</span>
       location_html:
         one: >-
-          <span class='govuk-!-font-weight-bold'>%{count}</span> job found near
+          <span class="govuk-!-font-weight-bold">%{count}</span> job found within
+          <span class='govuk-!-font-weight-bold'>%{radius}</span> miles of
```

Comment:
Again, happy for this to be a separate PR: shall we distinguish between "1 mile" and "n mile *s*"? We can use the `pluralize` method on the string 'miles', where `mile.pluralize(1)` gives 'mile' and `mile.pluralize(2)` gives 'miles'.